renderdoc/driver/vulkan/vk_resources.h

```diff
@@ -1097,6 +1097,14 @@ struct VkResourceRecord : public ResourceRecord
   ResourceId baseResource;
   ResourceId baseResourceMem;    // for image views, we need to point to both the image and mem
 
+  VkDeviceSize memOffset;
+  VkDeviceSize memSize;
```

Comment:
It's not an issue for this PR but I want to keep an eye on the size of `VkResourceRecord` and `ResourceRecord` in general. A lot of these structs get allocated during capture and some of the members are accessed with high frequency. In `VkResourceRecord` I have per-type cold data pointers, e.g. for buffers and images it's `ResourceInfo *resInfo` for lesser accessed data.

I think this is fine for now as this data will be accessed with high frequency. Plus there's definitely fat that can be trimmed with some refactors and/or moving data out of `ResourceRecord` that's only used in some APIs. More of a mental note that space in this struct is relatively precious.